azerothcore/azerothcore-wotlk
# Fix (db/creature) spire frostwyrm creature no longer duplicates in icc

## Changes Proposed:
-  remove the duplicate of the Spire Frostwyrm creature that just sits in the sky doing nothing in the ICC, right before the Gun Ship

## Issues Addressed:

- Closes 

## SOURCE:

![image](https://user-images.githubusercontent.com/122452427/230153619-bd008498-7f52-4ae9-a956-01aebfed9b9b.png)

## Tests Performed:

- 
- 


## How to Test the Changes:


1. go to the entrance just before the Gun Ship
2. look up and the dragon won't be there anymore
3. now only one dragon will come down and the copy will no longer idle on top

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
